README.md

```diff
@@ -111,22 +136,115 @@ The Customer Data Cloud Accelerator allows reading, working locally and deployin
 -   Web ScreenSets <a id="features-webscreensets"></a>
 -   Web SDK <a id="features-web-sdk"></a>
 
-### Local Live Preview <a id="features-local-preview"></a>
+### How to configure the use of features on the file cdc-accelerator.json
+
+On the cdc-accelerator.json file, there are two mandatory properties that the user has to fill, the "source" and "deploy".
```

Comment:
Use the same syntax to refer to files, cdc-accelerator.json or other names, source, deploy (see line 45 where the names are between ``)